ctapipe/tools/dl1_merge.py

```diff
@@ -0,0 +1,388 @@
+        help="Skip DL1/Event/Telescope and Simulation/Event/Telescope images in output",
+        default_value=False,
+    ).tag(config=True)
+    skip_simu_images = traits.Bool(
+        help="Skip Simulation/Event/Telescope images in output",
+        default_value=False,
+    ).tag(config=True)
+    skip_parameters = traits.Bool(
+        help="Skip DL1/Event/Telescope and Simulation/Event/Telescope parameters"
+        "in output",
+        default_value=False,
+    ).tag(config=True)
+    skip_broken_files = traits.Bool(
+        help="Skip broken files instead of giving error",
+        default_value=False,
+    ).tag(config=True)
+    overwrite = traits.Bool(help="Overwrite output file if it exists").tag(config=True)
+    progress_bar = traits.Bool(help="Show progress bar during " "processing").tag(
+        config=True
+    )
+    file_pattern = traits.Unicode(
+        default_value="*.h5", help="Give a specific file pattern for the" "input files"
+    ).tag(config=True)
+
+    parser = ArgumentParser()
+    parser.add_argument("input_files", nargs="*", type=Path)
+
+    aliases = {
+        "i": "MergeTool.input_dir",
+        "o": "MergeTool.output_path",
+        "p": "MergeTool.file_pattern",
+    }
+
+    flags = {
+        "skip-images": (
+            {"MergeTool": {"skip_images": True}},
+            "Skip DL1/Event/Telescope and Simulation/Event/Telescope images in output",
+        ),
+        "skip-simu-images": (
+            {"MergeTool": {"skip_simu_images": True}},
+            "Skip Simulation/Event/Telescope images in output",
+        ),
+        "skip-parameters": (
+            {"MergeTool": {"skip_parameters": True}},
+            "Skip DL1/Event/Telescope and Simulation/Event/Telescope parameters in output",
+        ),
+        "skip-broken-files": (
+            {"MergeTool": {"skip_broken_files": True}},
+            "Skip broken files instead of giving error",
+        ),
+        "overwrite": (
+            {"MergeTool": {"overwrite": True}},
+            "Overwrite output file if it exists",
+        ),
+        "progress": (
+            {"MergeTool": {"progress_bar": True}},
+            "Show a progress bar during event processing",
+        ),
+    }
+
+    def setup(self):
+        # prepare output path:
+        self.output_path = self.output_path.expanduser()
+        if self.output_path.exists():
+            if self.overwrite:
+                self.log.warning(f"Overwriting {self.output_path}")
+                self.output_path.unlink()
+            else:
+                self.log.critical(
+                    f"Output file {self.output_path} exists, "
+                    "use `--overwrite` to overwrite"
+                )
+                sys.exit(1)
+
+        PROV.add_output_file(str(self.output_path))
+
+        if self.skip_parameters is True and self.skip_images is True:
+            self.log.warning("Skip-parameters and skip-images are both set to True")
+            sys.exit(1)
```

Comment:
Someone could be interested in e.g. only extracting trigger times or simulation information. So maybe leave the warnings but for sure don't exit.